plugin/src/main/scala/org/scalafmt/sbt/ScalafmtPlugin.scala

```diff
@@ -62,6 +62,8 @@ object ScalafmtPlugin extends AutoPlugin {
       "Execute the scalafmtCheck task for all configurations in which it is enabled. " +
         "(By default this means the Compile and Test configurations.)"
     )
+    val scalafmtDetailedError =
+      settingKey[Boolean]("Log detailed error with stack trace, off by default")
```

Comment:
Not sure about English grammar :)

May be `Enables logging of detailed errors with stacktraces, disabled by default`

Reply:
It sounds better :)